### ISSUE TITLE
Put 3D printers in a server enclosure we've been offered

### ISSUE THREAD

**Issue by ajlennon** [DoESLiverpool/somebody-should#1373]
Hi all,

We've been offered this server enclosure which I reckon is big enough to take 2 3D printers. Had a quick chat with @amcewen who thinks this might be a good idea.

This is over in LSP at the moment. If we agree it's a good plan (i.e. reduce fire risk, maybe improve ABS printing) can somebody help me get it over to DoES? My boot won't fit this I don't think.

![pc box](https://user-images.githubusercontent.com/1537834/72741317-50d1ff80-3b9f-11ea-9c0f-2f8f54ad9ffa.jpeg)

![pc box 2](https://user-images.githubusercontent.com/1537834/72741328-53ccf000-3b9f-11ea-979d-b566101baa46.jpeg)

**Comment by ajlennon** [comment]
Can anybody help me with transportation on this? I think it's gonna be too big to get in my car

**Comment by goatchurchprime** [comment]
It would help if you gave dimensions so I could measure it for my boot or
roofrack

Or we get a trolley and push it down the road like the sewing machine
(preferred option).  Say when we can try.

**Comment by johnmckerrell** [comment]
I have the twin for the red skate/dolly at home, if I brought that in we might be able to wheel it all the way on the two of them? It’s probably too big for a cargo bike?

**Comment by amcewen** [comment]
Does anyone know where the red skate/dolly here is?  I couldn't find it on Saturday when we were moving the sewing machine, so @ajlennon @goatchurchprime and Zombie Chris had to carry it all the way down the street...

**Comment by johnmckerrell** [comment]
It was in the events room by the PA system, now it’s on the bottom shelf of the racking by the toilets.

**Comment by amcewen** [comment]
@ajlennon would we be able to pick it up tomorrow?  John C has hired a van for some sensor work we need to do, so we could possibly grab it tomorrow - around lunchtime maybe?

**Comment by ajlennon** [comment]
That would be great if possible thanks. I have to be in Knowsley for 2pm. I could meet you 12-1 or prep one of the guys in the office to meet you?

**Comment by ajlennon** [comment]
I've taken it down to the reception area in IC1 at LSP @amcewen. I have to head to Knowsley about 12 so if I'm not about Seimon will help out!

**Comment by amcewen** [comment]
Server rack retrieved from LSP and currently next to the bookcase-of-things in the main room until we work out where it should live longer term.

**Comment by johnmckerrell** [comment]
Indeed. Thanks to the transporting crew of John, John and John (and some guy named Adrian) 🤣

--

**Comment by ajlennon** [comment]
Well done guys
